src/btree_private.h

```diff
@@ -80,7 +80,7 @@ _Static_assert(offsetof(index_entry, key) == sizeof(index_entry),
  * *************************************************************************
  */
 #define MESSAGE_TYPE_BITS (2)
-_Static_assert(MESSAGE_TYPE_INVALID < (1ULL << MESSAGE_TYPE_BITS),
+_Static_assert(MESSAGE_TYPE_MAX_VALID_USER_TYPE < (1ULL << MESSAGE_TYPE_BITS),
                "MESSAGE_TYPE_BITS is too small");
```

Comment:
Given that we care about the number of bits used to represent the message type, I'm not terribly excited to add another type for static type-checking like this. We may in the future want to add another real type and then I think we may regret this, although I suppose we can undo it then.

Reply:
@ajhconway I am understanding you do not expect me to change any code here. We will leave as it is, and if necessary, it can be reworked in the future.